solr/core/src/test/org/apache/solr/util/BaseTestCircuitBreaker.java

```diff
@@ -193,6 +195,55 @@ public void testFakeCPUCircuitBreaker() {
     }
   }
 
+  public void testFakeLoadAverageCircuitBreaker() {
+    removeAllExistingCircuitBreakers();
+
+    CircuitBreaker circuitBreaker = new FakeLoadAverageCircuitBreaker();
+    LoadAverageCircuitBreaker loadAverageCircuitBreaker =
+        (LoadAverageCircuitBreaker) circuitBreaker;
+
+    loadAverageCircuitBreaker.setThreshold(75);
+
+    h.getCore().getCircuitBreakerRegistry().register(circuitBreaker);
```

Comment:
Lots of code duplication here across three different tests. Will factor out an assert method for these.